lightstep_carrier.proto

```diff
@@ -2,6 +2,8 @@ syntax = "proto3";
 
 package lightstep;
 
+option go_package = "lightsteppb";
```

Comment:
Is this the package that was already being used in Go?

Reply:
yup!